### DISCUSSION TITLE
combined of multiple button functions

### DISCUSSION THREAD

**Discussion by Fendo99** [MiczFlor/RPi-Jukebox-RFID#1864] [Q&A]
Hey  everyone,

i want to build an Phoniebox with just 2 buttons. An demo build on breadboard is already running, but i struggle with the control of the 2 buttons.
What i did until yet is: I connected 3 buttons.
1. button: shutdown / wake up
2. button: click for prev / hold for vol down
3. button: click for next / hold for vol up

But, like i said at the beginning, i just want to control all of that with just 2 buttons.
What i want is:
1. button: click for prev / hold for vol down
2. button: click for next / hold for vol up / if standby wake up
1. + 2. button: hold for shutdown

Did anyone do something like that? Or have an idea if it is possible?

**Comment by AlvinSchiller** [comment]
Something like that is only possible in v3 with the [TwinButton: Six function beast](https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/future3/main/documentation/builders/gpio.md#twinbutton-six-function-beast)
